code/modules/clothing/spacesuits/rig.dm

```diff
@@ -379,7 +382,10 @@
 	armor = list(melee = 60, bullet = 50, laser = 30, energy = 15, bomb = 35, bio = 100, rad = 50)
 	allowed = list(/obj/item/weapon/gun,/obj/item/ammo_box,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/melee/energy/sword,/obj/item/weapon/restraints/handcuffs,/obj/item/weapon/tank)
 	species_restricted = null
-	sprite_sheets = null
+	species_fit = list("Vox")

```

Comment:
Vox can already wear a blood-red hardsuit with no modification, so all this does is keep that functionality while applying a Vox-specific sprite once the suit's worn.